src/borg/testsuite/platform.py

```diff
@@ -75,154 +75,162 @@ def are_acls_working():
-        self.assert_equal(acl_use_local_uid_gid(b"group:root:rw-:0"), b"group:0:rw-")
-
-
-@unittest.skipUnless(sys.platform.startswith("darwin"), "macOS only test")
-@unittest.skipIf(fakeroot_detected(), "not compatible with fakeroot")
-class PlatformDarwinTestCase(BaseTestCase):
-    def setUp(self):
-        self.tmpdir = tempfile.mkdtemp()
-
-    def tearDown(self):
-        shutil.rmtree(self.tmpdir)
-
-    def get_acl(self, path, numeric_ids=False):
-        item = {}
-        acl_get(path, item, os.stat(path), numeric_ids=numeric_ids)
-        return item
-
-    def set_acl(self, path, acl, numeric_ids=False):
-        item = {"acl_extended": acl}
-        acl_set(path, item, numeric_ids=numeric_ids)
-
-    @unittest.skipIf(not are_acls_working(), "ACLs do not work")
-    def test_access_acl(self):
-        file = tempfile.NamedTemporaryFile()
-        file2 = tempfile.NamedTemporaryFile()
-        self.assert_equal(self.get_acl(file.name), {})
-        self.set_acl(
-            file.name,
-            b"!#acl 1\n"
-            b"group:ABCDEFAB-CDEF-ABCD-EFAB-CDEF00000000:staff:0:allow:read\n"
-            b"user:FFFFEEEE-DDDD-CCCC-BBBB-AAAA00000000:root:0:allow:read\n",
-            numeric_ids=False,
-        )
-        self.assert_in(
-            b"group:ABCDEFAB-CDEF-ABCD-EFAB-CDEF00000014:staff:20:allow:read", self.get_acl(file.name)["acl_extended"]
-        )
-        self.assert_in(
-            b"user:FFFFEEEE-DDDD-CCCC-BBBB-AAAA00000000:root:0:allow:read", self.get_acl(file.name)["acl_extended"]
-        )
-        self.set_acl(
-            file2.name,
-            b"!#acl 1\n"
-            b"group:ABCDEFAB-CDEF-ABCD-EFAB-CDEF00000000:staff:0:allow:read\n"
-            b"user:FFFFEEEE-DDDD-CCCC-BBBB-AAAA00000000:root:0:allow:read\n",
-            numeric_ids=True,
-        )
-        self.assert_in(
-            b"group:ABCDEFAB-CDEF-ABCD-EFAB-CDEF00000000:wheel:0:allow:read", self.get_acl(file2.name)["acl_extended"]
-        )
-        self.assert_in(
-            b"group:ABCDEFAB-CDEF-ABCD-EFAB-CDEF00000000::0:allow:read",
-            self.get_acl(file2.name, numeric_ids=True)["acl_extended"],
-        )
-
-
-@unittest.skipUnless(sys.platform.startswith(("linux", "freebsd", "darwin")), "POSIX only tests")
-class PlatformPosixTestCase(BaseTestCase):
-    def test_swidth_ascii(self):
-        self.assert_equal(swidth("borg"), 4)
-
-    def test_swidth_cjk(self):
-        self.assert_equal(swidth("バックアップ"), 6 * 2)
-
-    def test_swidth_mixed(self):
-        self.assert_equal(swidth("borgバックアップ"), 4 + 6 * 2)
+def get_linux_acl(path, numeric_ids=False):
+    item = {}
+    acl_get(path, item, os.stat(path), numeric_ids=numeric_ids)
+    return item
+
+
+def set_linux_acl(path, access=None, default=None, numeric_ids=False):
+    item = {"acl_access": access, "acl_default": default}
+    acl_set(path, item, numeric_ids=numeric_ids)
+
+
+@pytest.mark.skipif(not sys.platform.startswith("linux"), reason="linux only test")
```

Comment:
In a later PR `sys.platform.startswith(...)` could be replaced with the already computed `is_someplatform` from `borg.platformflags`.

Reply:
I could go ahead and make those changes to this file in this PR, if you would like.